docs-v2/components/Link.js

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import NextLink from 'next/link';
+import { window } from 'global';
+
+const versionRegex = [/^\/([0-9]+-[0-9]+-[0-9]+)\//, /^\/([0-9]+-[0-9]+-[0-9]+)$/];
+
+function getVersionHref(path, href) {
```

Comment:
Can you explain the rationale for this?
I'm having a hard time understanding what this exactly is doing / is for?

Reply:
Let's say we are browsing in https://storybook.js.org/3-2-9/help. Every link here should be related to the /3-2-9/ subdir. But the problem is that every link starts with '/' so it's actually relative to the root. In this case clicking on this link will open the wrong page (or will shot a wrong `window.location` that after refresh will open a wrong page). 
`getVersionHref` is looking for the `/x-x-x/` pattern and then returns the url that is base on this version. 
After that it's attached to the `as` parameter of the `next\link` that will just show the right url in the `window.location`

Reply:
gotcha!